src/content/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy.mdx

```diff
@@ -3,17 +3,17 @@ title: Mobile monitoring agents EOL policy
 metaDescription: Policies, start and end dates for support of New Relic mobile monitoring agent releases.
 ---
 
-The following are the specific policies and dates for support of our mobile monitoring agents: iOS/XCFramework, Android, and React Native. See our documentation about [general EOL policies](/docs/licenses/end-of-life/notification-changes-new-relic-saas-features-distributed-software/) for information about New Relic's overall end-of-life policy.
+The following are the specific policies and dates for support of our mobile monitoring agents: iOS (previously XCFramework), Android, and React Native. See our documentation about [general EOL policies](/docs/licenses/end-of-life/notification-changes-new-relic-saas-features-distributed-software/) for information about New Relic's overall end-of-life policy.
 
-## iOS/XCFramework [#ios-eol]
+## iOS [#ios-eol]
 
-The following are the specific policies and dates for support of our mobile monitoring agents for iOS/XCFramework. Any versions not listed in the following table are no longer supported. Please [update your XCFramework agent version](/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk) to the [latest release](/docs/release-notes/mobile-release-notes/xcframework-release-notes/).
+The following are the specific policies and dates for support of our iOS agent. Any versions not listed in the following table are no longer supported. Please [update your iOS agent version](/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk) to the [latest release](/docs/release-notes/mobile-release-notes/xcframework-release-notes/).
 
 <table>
   <thead>
     <tr>
       <th>
-        [iOS/XCFramework agent release](/docs/release-notes/mobile-release-notes/xcframework-release-notes)
+        [iOS agent release](/docs/release-notes/mobile-release-notes/xcframework-release-notes)
```

Comment:
Should we update the path here as well? `/docs/release-notes/mobile-release-notes/ios-agent-release-notes`

Reply:
Good catch, thank you!